Toolset/libraries/revidedeveloperextensionlibrary.livecodescript

```diff
@@ -998,5 +1006,46 @@ private function __revIDEDeveloperExtensionAddSpecifiedFilesToPackage pSource, p
       end if
    end if
    
+   # Add default script
+   if tError is empty and pDefaultScript is not empty then
+      revZipAddItemWithFile pArchive, "support/defaultscript.livecodescript", pDefaultScript
+      
+      if the result begins with "ziperr" then
+         __revIDEDeveloperExtensionSendWarning "couldn't add default script" && pDefaultScript
+      end if
+   end if
+   
    return tError
 end __revIDEDeveloperExtensionAddSpecifiedFilesToPackage
+
+on revIDEDeveloperExtensionEditDefaultScript pFolder
+   local tDefaultScript, tTargetStack
+   put pFolder & "/support/defaultscript.livecodescript" into tDefaultScript
+   
+   local tExpectedName
+   put sExtensionDetailsA[pFolder]["id"] & ".__DefaultScript" into tExpectedName
+   if there is a file tDefaultScript then
+      try
+         # If we have a valid stack, open it and edit the script
+         open stack tDefaultScript
+         put the long id of stack tDefaultScript into tTargetStack
+         revIDEEditScriptOfObject tTargetStack
```

Comment:
Is it necessary to add a check that `the name of tTargetStack is tExpectedName` here?